contracts/discovery/GNS.sol

```diff
@@ -754,4 +1015,16 @@ contract GNS is GNSV2Storage, GraphUpgradeable, IGNS, Multicall {
         require(_isPublished(subgraphData) == true, "GNS: Must be active");
         return subgraphData;
     }
+
+    function _versionExists(Version storage _version) internal view returns (bool) {
+        return _version.subgraphDeploymentID != "";
```

Comment:
I would do this check like `_version.subgraphDeploymentID != 0` to make it consistent with other empty checks